docs/en/api/metrics-query-expression.md

```diff
@@ -206,3 +206,22 @@ aggregate_labels(total_commands_rate, SUM)
 
 ### Result Type
 The ExpressionResultType of the aggregateLabels operation is TIME_SERIES_VALUES.
+
+## ViewAsSequence Operation
+ViewAsSequence operation represents the first not-null metric from the listing metrics in the given prioritized sequence(left to right). It could also be considered as a `short-circuit` of given metrics for the first value existing metric.
```

Comment:
Suggestion: make it clear that it does not only check `non-null`, but also check the `isEmptyValue`

Reply:
`isEmptyValue` actually `non-null`, because we were using `0` to represent null, which had been changed. So, this is `isEmptyValue` flag.